fspiop-api/documents/API-Definition_v1.1.1.md

```diff
@@ -2264,14 +2264,14 @@ The commit notification is sent in the request **PATCH /transfers/**_{ID}_ from
 
 In case of a validation failure (not schema validation) or another processing error during the processing of a **PUT /transfers/**_{ID}_ callback, a **PATCH /transfers/**_{ID}_**/error** notification can be sent from the Switch to the Payee FSP to indicate of the failure. This notification is sent in cases where the Payee FSP sends the _transferState_ as committed.
 
-[Figure 56a](#figure-56a) shows an example in which the validation in the Switch failed due to some reason, for example a business rule validation caused a failure in the Switch. This is the same example as in [Figure 53](#figure-53), but the Payee FSP receives a commit error notification so that the Payee FSP can take remedial action. The Payee FSP would need to use an internal reversal of the transaction (or similar).
+[Figure 56](#figure-56a) shows an example in which the validation in the Switch failed due to some reason, for example a business rule validation caused a failure in the Switch. This is the same example as in [Figure 53](#figure-53), but the Payee FSP receives a commit error notification so that the Payee FSP can take remedial action. The Payee FSP would need to use an internal reversal of the transaction (or similar).
```

Comment:
I would like to have a new section for this text, as it should not be part of "6.7.2.6 Commit Notification". Suggesting to insert "6.7.2.7 Commit Error Notification" to discern the difference between the two concepts. Preferably also with some introductory text that this is an optional feature that could be used in case normal Commit Notifications cannot be used.

This new section would then move "6.7.2.7 Refunds" to 6.7.2.8 and "6.7.2.8 Interledger Payment Request" to 6.7.2.9...

Reply:
updated..